tutorials/define_and_compile_model.py

```diff
@@ -0,0 +1,181 @@
+"""
+Quick Start - Compile and Deploy Resnet-18 Model
```

Comment:
Quick Start - End-to-End Tutorial for Compiling and Deploying the Resnet-18 Model to Multiple Hardware Targets

Reply:
This title is long and it will go out of sphinx gallery box, which makes website look not very clean. How about "Quick Start - End-to-End Tutorial for NNVM/TVM Pipeline"?

Reply:
Sounds good.